docs/README.md

```diff
@@ -1,3 +1,13 @@
-This directory is for Markdown documentation.
+<!-- 
+Use this file to provide a index to the documents.
```

Comment:
```suggestion
Use this file to provide an index to the documents.
```

:-)

Reply:
@garethsb I've added a lot more instructions in the comment.

Reply:
PS the comment will appear in the final HTML.  There are ways of getting round that but they cause linting issues.  But I would expect spec authors to delete the comment anyway.